samples/03_org_administrators/validate_item_metadata.ipynb

```diff
@@ -48,7 +48,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        "snippet":"Report on item attributes from aPI"
</pre><p>little typo in snippet - API</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Fixed!</p>

Reply:
Fixed!